docs/writing-tests/testdriver-extension-tutorial.md

```diff
@@ -95,16 +105,24 @@ window.test_driver_internal.set_element_rect = function(x, y, width, height) {
 
 The `create_action` helper function does the heavy lifting of setting up a postMessage to the wptrunner internals as well as returning a promise that will resolve once the call is complete.
 
+The action's `name` is important and will be used later when defining the corresponding Python action representation. Keep this name in mind for the next steps.
+
+#### WebDriver BiDi
+
+For actions related to WebDriver BiDi, the `name` should follow the format `bidi.{MODULE_NAME}.{COMMAND}`, for example, [`bidi.session.subscribe`](https://github.com/web-platform-tests/wpt/blob/107c5fc03139b3247920a0f40983bd1fe4d1fac2/tools/wptrunner/wptrunner/testdriver-extra.js#L202).
+
+### Protocol part
+
 Next, this is passed to the executor and protocol in wptrunner. Time to switch to Python!
 
-### [tools/wptrunner/wptrunner/executors/protocol.py](https://github.com/web-platform-tests/wpt/blob/master/tools/wptrunner/wptrunner/executors/protocol.py)
+To add this command, you'll need to create a corresponding protocol part class in [tools/wptrunner/wptrunner/executors/protocol.py](https://github.com/web-platform-tests/wpt/blob/master/tools/wptrunner/wptrunner/executors/protocol.py).
 
 ```python
 class SetWindowRectProtocolPart(ProtocolPart):
     """Protocol part for resizing and changing location of window"""
     __metaclass__ = ABCMeta
 
-    name = "set_window_rect"
+    name = "set_window_rect_protocol_part"
```

Comment:
Actually this can be any string, and the protocol part will be put to the property with this name to the final `protocol` implementation. So to avoid confusion, I changed the name.

Reply:
https://github.com/web-platform-tests/wpt/pull/49168/files#r1842313203

Reply:
As mentioned, this part (the classic one) is outdated and is not aligned with the current implementation, so I adjusted the path to avoid confusion between those 2 string literals.

Reply:
I checked the current implementation and I see the pattern that the name matches the part name, e.g., in TestharnessProtocolPart, ClickProtocolPart, AccessibilityProtocolPart, ActionSequenceProtocolPart and others. I do not think we should encourage "_protocol_part" in the name.

Reply:
Do you have a better name in mind, as `set_window_rect` is confusing and interferes with the action name?